sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/src/Azure.WebJobs.Extensions.Storage.Blobs.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs" />
+    <PackageReference Include="Microsoft.Extensions.Azure" VersionOverride="1.1.0-alpha.20201021.1" />
```

Comment:
If you are depending on an unshipped version of this package it is probably better to keep it as a ProjectReference instead of pining to a given nightly build. After the version you need ships then switch to package reference.

Reply:
This reference is across service directories so the pack doensn't work correctly.